source/board/lpc55s26_bl.c

```diff
@@ -0,0 +1,101 @@
+/**
+ * @file    lpc5516_bl.c
+ * @brief   board ID and meta-data for the hardware interface circuit (HIC) based on the NXP LPC5516
```

Comment:
It should be `NXP LPC5526`

Reply:
Same as above, we've renamed it to LPC55S26 instead.